### ISSUE TITLE
Make file failed

### ISSUE THREAD

**Issue by JarvisKevin** [sebgao/LIP#1]
Hi, my name is Jarvis, a beginner of DL. I found I failed to make the file using "make train-lip_resnet50". Is a train-lip_resnet50.py needed? Thank you.

**Comment by sebgao** [comment]
`train-lip_resnet50.py` is not a need. Alternatively, you can explicitly run the code without the `Makefile`:
```
CUDA_VISIBLE_DEVICES=0,1,2,3,4,5,6,7 python main.py --dist-url 'tcp://127.0.0.1:12004' --dist-backend 'nccl' --multiprocessing-distributed --world-size 1 --rank 0 ~/dataset/imagenet-cls/ --module lip_resnet --arch resnet50 -b 256 --num_gpus 8
```

By the way, what's the error message?

**Comment by JarvisKevin** [comment]
I have only two GPU, so I have reseted the makefile like this![image](https://user-images.githubusercontent.com/39183050/63003351-87964200-beaa-11e9-8862-de010430ced9.png)
and after I run 'make train-lip_resnet50', then i got this error.
![image](https://user-images.githubusercontent.com/39183050/63003426-aeed0f00-beaa-11e9-8100-e9cb5283418f.png)

**Comment by sebgao** [comment]
Oops,  `-b` option stands for *total* batch size. If you have 2 GPUs, you should change `-b` 256 to 256/4=64 and lower learning rate `--lr` to 0.1/4=0.025 (linear scaling). Note that the training will take ~7 days on only two GPUs.

**Comment by JarvisKevin** [comment]
My god. what type of GPUs you used?

**Comment by sebgao** [comment]
Titan XP. Actually, you can download pretrained models if you want to fine-tune them on other tasks or evaluate them on ImageNet val set.

**Comment by JarvisKevin** [comment]
Ok, Thank you for your suggestion. Looking forward to your later code.
